design/gdd.md

```diff
@@ -1028,8 +1028,8 @@ Takes the [used_glass](items/used_glass.md) from [rebeca](characters/rebeca.md).
 
 #### PUZZLE 3.6: Comparing fingerprints
 
-- Use the [makeshift_fingerprint_kit](items/makeshift_fingerprint_kit.md) with [used_glass](items/used_glass.md) to get [rebeka_fingerprints](items/rebeka_fingerprints.md)
-- Use [rebeka_fingerprints](items/rebeka_fingerprints.md) with [fingerprint_from_hook](items/fingerprint_from_hook.md)
+- Use the [makeshift_fingerprint_kit](items/makeshift_fingerprint_kit.md) with [used_glass](items/used_glass.md) to get [rebeka_fingerprints](rebeca_fingerprints.md)
```

Comment:
Seems as if I myself wrote Rebecas name wrong. 😆 

However, the items/ subpath is missing in the link there. It should be "(items/reebca_fingerprints.md)"